commands/debug.js

```diff
@@ -56,6 +56,31 @@ if(args[0].toLowerCase()==="quest"){
   message.channel.send("This creature can't do quests.");
   return;
 }
+if(args[0].toLowerCase()==="capgrist"){
```

Comment:
I would probably tweak this command to operate on a targeted ping, so that a DM can do it for a player (all debug commands are gated to only DMs). Also, adding a check near the beginning so that if the allData function returns NONE (as it's targeting an NPC who doesn't have grist) it should return, because a sburbMap.get on an NPC will crash, should use allData as well.

Reply:
That does raise one question I've had, actually: what's the functional difference between "get" and "allData"?

Reply:
using  xMap.get and xMap.set is the usual way you read and write to Enmaps. Only issue with this is since the data rework for underlings and NPCS, all of their data is stored in npcdata, while a player's is split among playerdata, sburbdata, and userdata. charcall is a series of functions I made that automatically sorts out what kind of data it's looking for, and queries the right databases for it, as well as returning the string "NONE" instead of an undefined, or even worse, a crash, when certain data isn't found.